src/tracker/amplitude.js

```diff
@@ -26,6 +26,7 @@ export class AmplitudeTracker extends BaseTracker {
       debug("initialize Amplitude with Config %o", this.amplitudeConfig)
       this.getTracker().init(this.apiKey, null, this.amplitudeConfig)
     }
+    this.pushToDataLayer = options.pushToDataLayer === false ? false : true
```

Comment:
You can turn the boolean short hand condition into an instantly boolean transform by:

```suggestion
    this.pushToDataLayer = !!options.pushToDataLayer
```

Reply:
Resolved, the suggestion does not work if the option is undefined because we need to have a true value